README.md

```diff
@@ -16,3 +16,23 @@ copyright and related rights in the work worldwide are waived through
 the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/publicdomain/zero/1.0/).
 
 For more information, see [license](LICENSE.md).
+
+
+## Deployment
+
+Refer to [docs.18f.gov](https://docs.18f.gov/getting-started/setup/) for getting set up with Cloud Foundry
```

Comment:
maybe reference which org a person needs to be added to in order to deploy `dolores`?